scripts/scil_validate_and_correct_bvecs.py

```diff
@@ -0,0 +1,119 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Validate and correct b-vectors using a fiber coherence index as presented
+in Schilling et al, 2019. The script takes as input the principal direction(s)
+at each voxel, the b-vectors and the fractional anisotropy map and outputs
+a corrected b-vectors file.
+
+A typical pipeline could be:
+>>> scil_compute_dti_metrics.py dwi.nii.gz bval bvec --not_all --fa fa.nii.gz
+    --evecs peaks.nii.gz
+>>> scil_validate_and_correct_bvecs.py bvec peaks_v1.nii.gz fa.nii.gz bvec_corr
+
+Note that peaks_v1.nii.gz is the file containing the direction associated
+to the highest eigenvalue at each voxel.
+
+It is also possible to use a file containing multiple principal directions per
+voxel, given that the amplitude of each direction is also given with the
+argument --peaks_vals.
+"""
+
+import argparse
+import logging
+import numpy as np
+import nibabel as nib
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist, add_verbose_arg)
+from scilpy.io.image import get_data_as_mask
+from scilpy.reconst.fiber_coherence import compute_fiber_coherence_table
+from dipy.io.gradients import read_bvals_bvecs
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_bvecs', help='Path to bvecs file.')
```

Comment:
please put the help on a second line. Just for homogeneity. Check also others script. 
Example: in_bvec instead of in_bvecs, in_FA instead of in_fa.